### PR TITLE
Make app translatable

### LINKED ISSUE #10: Make app translatable
I can handle this once #7 will be merged to avoid conflicts :smile:

Then I will be happy to add French translation.

### PR DESCRIPTION
**Pull request by NathanBnm**
This PR closes #10

### CODE REVIEW

**Comment by NathanBnm** [comment]
This will close #10

File: po/com.github.rajkumaar23.badgie.pot [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# SOME DESCRIPTIVE TITLE.
```
Comment (rajkumaar23) [inline po/com.github.rajkumaar23.badgie.pot:1]: Can this be a more appropriate title ?

File: po/extra/extra.pot [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# SOME DESCRIPTIVE TITLE.
```
Comment (rajkumaar23) [inline po/extra/extra.pot:1]: And here too..

**Review by rajkumaar23 — CHANGES_REQUESTED** [review]

**Comment by NathanBnm** [comment]
@rajkumaar23 these are auto-generated files, when adding languages the appropriate header will be generated automatically. For example in French it will be:

```
# French translations for com.github.rajkumaar23.badgie package.
# Copyright (C) 2020 THE com.github.rajkumaar23.badgieS COPYRIGHT HOLDER
# This file is distributed under the same license as the com.github.rajkumaar23.badgie package.
# Nathan Bonnemains (@NathanBnm), 2020.
#
msgid ""
msgstr ""
"Project-Id-Version: com.github.rajkumaar23.badgie\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-06-15 09:50+0100\n"
"PO-Revision-Date: 2020-06-15 09:50+0100\n"
"Last-Translator: Nathan Bonnemains (@NathanBnm)\n"
"Language-Team: Français\n"
"Language: fr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
```

**Comment by NathanBnm** [comment]
Translation files are auto-generated by running

```
ninja com.github.rajkumaar23.badgie-update-po
```

for main translation and

```
ninja extra-update-po
```

for metadata translation

The `.pot` file is just a template file used by the generator to create the appropriate `.po` translation  file for each language code added in the `LINGUAS` file.

**Comment by rajkumaar23** [comment]
> Translation files are auto-generated by running
> 
> ```
> ninja com.github.rajkumaar23.badgie-update-po
> ```
> 
> for main translation and
> 
> ```
> ninja extra-update-po
> ```
> 
> for metadata translation
> 
> The `.pot` file is just a template file used by the generator to create the appropriate `.po` translation file for each language code added in the `LINGUAS` file.

Okay got it 👍🏼

**Review by rajkumaar23 — APPROVED** [review]

**Pull request merged**